### PR TITLE
Implement distribute coins in coin creator

### LINKED ISSUE #1468: Tool for genesis coin distribution
Generally the steps involved in distribution of coins are: 

1. Generate `n` distribution addresses, preferably `100` and substitute them here https://github.com/skycoin/skycoin/blob/develop/src/visor/distribution.go#L95

2. uncomment this code https://github.com/skycoin/skycoin/blob/71d24a0fe6d16e2a4ccb6833c0d0734c60b468e6/cmd/skycoin/skycoin.go#L799-L804 and set the genesis address secret key here

3. comment out the following lines https://github.com/skycoin/skycoin/blob/94523128669bab20389b558711f26b61dbc0c927/cmd/skycoin/skycoin.go#L791-L794

4. uncomment this https://github.com/skycoin/skycoin/blob/71d24a0fe6d16e2a4ccb6833c0d0734c60b468e6/cmd/skycoin/skycoin.go#L724-L732 and replace with
    ```
    time.Sleep(5)
    tx := InitTransaction()
    _, _, err = d.Visor.InjectTransaction(tx)
    if err != nil {
        log.Panic(err)
    }
   ```
5. Run the node to make the distribution transaction
6. Comment out the distribution code( make sure to remove the `seckey` 

## Task
- create a cli tool to use the `seckey` and send out the genesis coins to `100` random addresses, store the seeds of those addresses in a text file.

Steps:
1. get address of genesis block from seckey 
2. get uxid of that address
3. build a transaction with 100 outputs
4. inject the transaction

### PR DESCRIPTION
**Pull request by therealssj**
Fixes #1468 

Changes:
- implement coin distribution in coin creator tool

Does this change need to mentioned in CHANGELOG.md?
yes

### CODE REVIEW

**Comment by therealssj** [comment]
There is one thing though
you need to run the create command again with updated `fiber.toml` with the `genesis sig` which is the output of this command

File: cmd/newcoin/newcoin.go [hunk 1]
```diff
@@ -203,6 +210,109 @@ func createCoinCommand() cli.Command {
...
+			cmd.Wait()
+
+			log.Infof("genesis sig: %s", genesisSig)
+
+			return nil
+		},
+	}
+}
+
+func copyAndCapture(w io.Writer, r io.Reader) ([]byte, error) {
```
Comment (iketheadore) [inline cmd/newcoin/newcoin.go]: Remove this function if it's not being used.

Comment (therealssj) [reply cmd/newcoin/newcoin.go]: oh haha missed that

**Comment by therealssj** [comment]
@iketheadore `distributecoins` distributes coins to distribution addresses
https://github.com/skycoin/skycoin/pull/1726/files#diff-20f0c9dd309d214bd939e0f8f07b496bR205
calling `InitTransaction` here which distributes the coins

**Pull request closed without merging**